packages/stryker-api/src/report/ScoreResult.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Represents a score result of a file or directory
+ */
+interface ScoreResult {
+
+  /**
+   * The file or directory name of this score result node
+   */
+  readonly name: string;
+  
+  /**
+   * Any child directory/file score result nodes
+   * If this score result represents a file, the length will be 0
+   */
+  readonly childResults: ReadonlyArray<ScoreResult>;
```

Comment:
We won't add `isDirectory()` or `isFile()` until we have to know the difference between a file and a directory in a reporter.